app/assets/stylesheets/govuk_publishing_components/components/_search-with-autocomplete.scss

```diff
@@ -0,0 +1,197 @@
+
+    &--visible {
+      display: block;
+    }
+
+    &--hidden {
+      display: none;
+    }
+
+    &--overlay {
+      position: absolute;
+      left: 0;
+      top: 100%;
+      z-index: 100;
+
+      box-shadow: rgba(0, 0, 0, 0.2569) 0 2px 6px;
+    }
+
+    &--inline {
+      position: relative;
+    }
+  }
+
+  .autocomplete__option {
+    display: block;
+
+    cursor: pointer;
+
+    @include govuk-font(19);
+
+    // Ensure only the parent option receives pointer events
+    & > * {
+      pointer-events: none;
+    }
+
+    // Hover styling applies to both the currently keyboard-selected option and possibly
+    // additionally another option that is hovered over by mouse. Only the keyboard-selected option
+    // gets focus color applied to its text.
+    &--focused,
+    &:hover,
+    &:focus-visible {
+      background-color: govuk-colour("light-grey");
+      outline: none;
+    }
+
+    &:focus-visible {
+      .autocomplete__suggestion-text {
+        background-color: $govuk-focus-colour;
+      }
+    }
+
+    .autocomplete__option-wrapper {
+      display: flex;
+      align-items: center;
+      margin: 0 govuk-spacing(2);
+      padding: govuk-spacing(1) 0;
+
+      border-bottom: 1px solid govuk-colour("mid-grey");
+    }
+
+    &:last-child .autocomplete__option-wrapper {
+      border-bottom: 0;
+    }
+
+    // Accessible Autocomplete's iOS screenreader inset has broken CSS which hasn't been fixed
+    // upstream, and means that its text is not just visible to screenreader users, but displayed
+    // for everyone. This span is added dynamically only on iOS and not given a class, so we need to
+    // target it in a roundabout way and make it invisible to non-screenreader users.
+    & > span {
+      clip: rect(0 0 0 0);
+      clip-path: inset(50%);
+      height: 1px;
+      overflow: hidden;
+      position: absolute;
+      white-space: nowrap;
+      width: 1px;
+    }
+  }
+
+  .autocomplete__suggestion-icon {
+    width: $input-size;
+    height: $input-size;
+
+    flex: none;
+
+    background-image: url("govuk_publishing_components/icon-autocomplete-search-suggestion.svg");
+    background-repeat: no-repeat;
+  }
+
+  .autocomplete__suggestion-text {
+    font-weight: bold;
+
+    mark {
+      font-weight: normal;
+      background: none;
+    }
+  }
+
+  // Tweak the look and feel for the autocomplete in large mode
+  .gem-c-search--large {
+    .autocomplete__menu {
+      @include enhance-autocomplete-menu-width($large-input-size);
+    }
+
+    .autocomplete__option-wrapper {
+      padding: govuk-spacing(1) 0;
+    }
+
+    .autocomplete__option {
+      height: $large-input-size;
+    }
+  }
+
+  // Fix top border styling on "borderless" search input when rendered on a GOV.UK blue background
+  .gem-c-search--on-govuk-blue {
+    .autocomplete__menu {
+      border-top: 1px solid $govuk-border-colour;
+    }
+  }
+
+  // High contrast mode adjustments
+  @media (forced-colors: active) {
```

Comment:
No, they're not imported by default - we do want some elements of it but we style the menu and options quite differently for the most part, so it was easier to "adopt" the CSS and change it to our desired styling.

For the forced colours, I've tidied them up a bit removing special casing for the most outdated Grade C browsers, and I'm going to amend them a bit more following today's DAC audit.

We may be able to get away with less styling in forced colors mode, but we do need to adapt the styling a little as otherwise it currently looks like this:
<img width="976" alt="image" src="https://github.com/user-attachments/assets/4678e259-a7a3-440d-9b6c-19955e5751da">